### ISSUE TITLE
1Password SSH-Agent Support

### ISSUE THREAD

**Issue by jarrah31** [satiromarra/vscode-sftp#20]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [x] No.

**Is your feature request related to a problem? Please describe.**
I would like to be able to use the 1Password SSH-Agent with SFTP on MacOS. 
I can see you have forked this project and have been fixing issues since Natizyskunk's last release, so I was hoping you may know what this problem could be please?

**Describe the solution you'd like**
I have tried the following config:
```
{
    "name": "Docker",
    "host": "192.168.1.32",
    "protocol": "sftp",
    "port": 22,
    "username": "server",
    "remotePath": "/home/server/docker",
    "uploadOnSave": true,
    "useTempFile": false,
    "openSsh": false,
    "agent": "~/Library/Group Containers/XXXXXXX.com.1password/t/agent.sock",
    "sshConfigPath": "~/.ssh/config"
}
```

**Describe alternatives you've considered**
The 1Password agent works with VSCode's Remote SSH and the Mac's Terminal client, but within SFTP it gives the following error.
```
[2023-03-23 19:14:45] [error] Error: [192.168.1.32]: Failed to connect to agent
	at Client.<anonymous> (/Users/me/.vscode/extensions/satiromarra.code-sftp-1.17.9/dist/extension.js:2:291992)
	at Client.emit (node:events:538:35)
	at /Users/me/.vscode/extensions/satiromarra.code-sftp-1.17.9/node_modules/ssh2/lib/client.js:965:22
	at /Users/me/.vscode/extensions/satiromarra.code-sftp-1.17.9/node_modules/ssh2/lib/agent.js:1042:15
	at process.processTicksAndRejections (node:internal/process/task_queues:78:11) download /Users/me/Library/Mobile Documents/com~apple~CloudDocs/configuration.yaml
```

**Does this project help you?**
- [x] Not quite yet unfortunately

**Comment by satiromarra** [comment]
@jarrah31 Try with the absolute path to agent:
 ```json
"agent": "/Users/me/Library/Group Containers/XXXXXXX.com.1password/t/agent.sock"
```

I don't use 1password, I just installed it but I don't know how to set it up to use it with the agent.

**Comment by satiromarra** [comment]
@jarrah31 I have managed to configure 1password and the agent and I have been able to use it without any problem.
First:
```json
"agent": "/Users/me/Library/Group Containers/XXXXXXX.com.1password/t/agent.sock"
```
Later:
Add your private key in 1password (located in /Users/me/.ssh/id_rsa)

Next:
Open the connection, In my case, it opens a macOS window to validate (password or touch id) the connection and then it will show you the sftp content.

I hope I have helped you.
